index.html

```diff
@@ -1125,12 +1086,11 @@ <h2>
         integrity="sha512-bPs7Ae6pVvhOSiIcyUClR7/q2OAsRiovw4vAkX+zJbw3ShAeeqezq50RIIcIURq7Oa20rW2n2q+fyXBNcU9lrw=="
         crossorigin="anonymous"></script>
     <!-- Loading the javaScript at the end of the page -->
-    <script src="js/bootstrap.js" type="text/javascript">
-    </script>
-    <script src="js/navScroll.js" type="text/javaScript"></script>
+    <script src="js/bootstrap.js"></script>
+    <script src="js/navScroll.js"></script>
 
     <!-- OWl Carousel -->
-    <script type="text/javascript">
```

Comment:
is removing type fixes something?

Reply:
It was giving a warning that script doesn't require type in HTML5.